modules/dnn/src/torch/torch_importer.cpp

```diff
@@ -1201,12 +1201,12 @@ struct TorchImporter
                     }
                 }
 
-                module->params.set("pool_k_h", poolingLayer.second->params.get<int>("kernel_h"));
-                module->params.set("pool_k_w", poolingLayer.second->params.get<int>("kernel_w"));
-                module->params.set("pool_stride_h", poolingLayer.second->params.get<int>("stride_h"));
-                module->params.set("pool_stride_w", poolingLayer.second->params.get<int>("stride_w"));
-                module->params.set("pool_pad_h", poolingLayer.second->params.get<int>("pad_h"));
-                module->params.set("pool_pad_w", poolingLayer.second->params.get<int>("pad_w"));
+                module->params.set("kernel_h", poolingLayer.second->params.get<int>("kernel_h"));
+                module->params.set("kernel_w", poolingLayer.second->params.get<int>("kernel_w"));
+                module->params.set("stride_h", poolingLayer.second->params.get<int>("stride_h"));
+                module->params.set("stride_w", poolingLayer.second->params.get<int>("stride_w"));
+                module->params.set("pad_h", poolingLayer.second->params.get<int>("pad_h"));
+                module->params.set("pad_w", poolingLayer.second->params.get<int>("pad_w"));
```

Comment:
Let's keep them as is. I think that there is no critical difference for now.

Reply:
tensorflow's setKSize use ‘’kernel_h‘’ and ‘’kernel_w‘’, setStrides use ‘’stride_h‘ and ’stride_w‘. This is somewhat different from the variable name of tensorflow.

Reply:
`setKSize` sets parameters to `poolingLayer.second->params`. You can use any names inside the layer.

Reply:
My intention is to reuse setKSize and setStrides method.